Reorganize documentation content (#984)
ROCm/roc-optiq: docs/how-to/view-trace.rst

Reviewer: ```suggestion
.. |make| image:: ../images/make-selection.png
.. |remove| image:: ../images/remove-time.png

```

- Click |eye| to show or hide tracks. Hiding a higher-level element automatically hides all underlying queues, streams, and events.
- Click |scroll| to go to the track in the :ref:`timeline`. 

> - Right click to chose sidebar context menu options: **Go To Track**, **Hide Track**, **Show/Hide All But This Track**, **Show/Hide Selected Tracks**.

Reviewer: ```suggestion
- Right click to choose sidebar context menu options: **Go To Track**, **Hide Track**, **Show/Hide All But This Track**, **Show/Hide Selected Tracks**.
```

For sample counter tracks, pills showing standard deviation and average display next to the counter label. These can be shown/hidden using the **Track Options** menu.  
Additionally, min and max value pills can also be shown/hidden. 

> When a counter track is expanded to sufficient height, the Y-axis now shows incremental scale tick labels between the min and max values, making intermediate values easier to read without opening **Track Details**.

Reviewer: ```suggestion
When a counter track is expanded to sufficient height, the Y-axis now shows incremental scale tick labels between the minium and maxium values, making intermediate values easier to read without opening **Track Details**.
```

For sample counter tracks, pills showing standard deviation and average display next to the counter label. These can be shown/hidden using the **Track Options** menu.  
Additionally, min and max value pills can also be shown/hidden. 

> When a counter track is expanded to sufficient height, the Y-axis now shows incremental scale tick labels between the minium and maxium values, making intermediate values easier to read without opening **Track Details**.

Reviewer 2: ```suggestion
When a counter track is expanded to sufficient height, the Y-axis shows incremental scale tick labels between the minimum and maxium values, making intermediate values easier to read without opening **Track Details**.
```

- Clicking on a row in the search results will bring the selected event into view on the :ref:`timeline`. 
- Clicking **X** clears the search results.
- The search can match multiple substrings at once. Multiple search tokens must be surrounded by quotation marks without spaces (for example: ``“term1””term2”``).

> Save trace selection (Trim trace)

Reviewer: ```suggestion
Save trace selection (trim trace)
```
